devnotes/undocumented_registers.md

```diff
@@ -0,0 +1,1062 @@
+# What this document is about?
+This document is an attempt to gather information about undocumented registers
+and registers that code analysis proofed to work diffrently than described.
+
+**P9N2_MCS_PORT02_MCPERF0 0x5010823**
+| Bit range | Name      | Assigned value | Description |
```

Comment:
The description of a table. E.g. what is `assigned value`? Default value or something what we set maybe?

Reply:
To clarify the document, I have renamed the column to `Value assigned in code`.